cmake/EthCompilerSettings.cmake

```diff
@@ -153,8 +153,10 @@ if (("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU") OR ("${CMAKE_CXX_COMPILER_ID}" MA
 
 # The major alternative compiler to GCC/Clang is Microsoft's Visual C++ compiler, only available on Windows.
 elseif (DEFINED MSVC)
+	# Remove NDEBUG from RELWITHDEBINFO (to enable asserts)
+	string(REPLACE "/DNDEBUG" " " CMAKE_CXX_FLAGS_RELWITHDEBINFO "${CMAKE_CXX_FLAGS_RELWITHDEBINFO}")
```

Comment:
You could also mention that `CMAKE_CXX_FLAGS_RELWITHDEBINFO` for GCC/Cland does not include `NDEBUG`.
I did not expect that difference actually.